extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/attribute/ResponseTimeAttribute.java

```diff
@@ -30,6 +34,7 @@ public ResponseTimeAttribute(TimeUnit timeUnit) {
     public String readAttribute(RoutingContext exchange) {
         Long requestStartTime = exchange.get(VertxHttpRecorder.REQUEST_START_TIME);
         if (requestStartTime == null) {
+            log.warn("Response time for request cannot be computed since the request start time hasn't been recorded");
```

Comment:
I think I would rather do it in the constructor by checking the config rather than doing it per request.

And yes, I would still make it log the message once as we could probably have multiple log patterns referencing this.

Also I would make the warning explain how to fix it.

Does it make sense?

Reply:
Hello @gsmet, yes that makes sense. I'm offline for a few days. I'll update this PR as soon as I'm back.

Reply:
@jaikiran friendly ping :).

Reply:
Oops, I completely forgot this one. I'll refresh this PR shortly :)